### PR TITLE
add grafana to the global hub

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>

### CODE REVIEW

File: operator/pkg/controllers/hubofhubs/manifests/grafana/clusterrolebinding.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    name: multicluster-global-hub-grafana
+  name: multicluster-global-hub-grafana
+  namespace: {{.Namespace}}
```
Comment (morvencao) [inline operator/pkg/controllers/hubofhubs/manifests/grafana/clusterrolebinding.yaml:7]: namespace for `ClusterRoleBinding`? delete this?

File: operator/pkg/controllers/hubofhubs/manifests/grafana/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  selector:
+    matchLabels:
+      name: multicluster-global-hub-grafana
+  template:
+    metadata:
+      labels:
+        name: multicluster-global-hub-grafana
+    spec:
+      affinity:
+        podAntiAffinity:
```
Comment (morvencao) [inline operator/pkg/controllers/hubofhubs/manifests/grafana/deployment.yaml:19]: do we really want podAntiAffinity and  podAffinity, you have set the repolicas to `1` and it is not customized.

File: operator/pkg/controllers/hubofhubs/manifests/grafana/service.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    name: multicluster-global-hub-grafana
+  name: multicluster-global-hub-grafana
+  namespace: {{.Namespace}}
+  annotations:
+    service.alpha.openshift.io/serving-cert-secret-name: multicluster-global-hub-grafana-tls
```
Comment (morvencao) [inline operator/pkg/controllers/hubofhubs/manifests/grafana/service.yaml:9]: change annotatation key to `beta`:
```
service.beta.openshift.io/serving-cert-secret-name: multicluster-global-hub-grafana-tls
```

File: operator/pkg/controllers/hubofhubs/manifests/grafana/deployment.yaml (hunk 2 above)
Comment (yanmxa) [reply operator/pkg/controllers/hubofhubs/manifests/grafana/deployment.yaml:19]: I thought about it, if we want to provide a Global View at TP, we should need a more reliable service. So I set the replicas to 2, and the podAntiAffinity or PodAffinity should work.

Comment (morvencao) [reply operator/pkg/controllers/hubofhubs/manifests/grafana/deployment.yaml:19]: ok

**Review by morvencao — COMMENTED** [review]
/lgtm

**Review by morvencao — COMMENTED** [review]
/lgtm

**Pull request merged**